test_data/asm/wrap_gl.asm

```diff
@@ -0,0 +1,123 @@
+// Wraps an arbitrary field element into a u32, on the Goldilocks field.
+machine WrapGL(RESET, operation_id) {
+
+    operation wrap<0> in_acc -> output;
+
+    constraints {
+
+        // Latch and operation ID
+        col fixed RESET(i) { ((i % 8) - 7) == 0 };
```

Comment:
Is this `[0,0,0,0,0,0,0,1]*`? I wonder if it's clearer with the array, or maybe introduce a macro `last_in_block_of_size`?